kratos/tests/test_model.py

```diff
@@ -39,6 +39,10 @@ def test_model(self):
         #self.assertEqual(model["Main.Outlet"].Info(), outlet.Info() )
         #self.assertEqual(model["Main.Outlet.aaa"].Info(), aaa.Info() )
 
-        
+        self.assertTrue(model.HasModelPart("Main"))
+        self.assertTrue(model.HasModelPart("Main.Outlet"))
+        self.assertFalse(model.HasModelPart("Outlet"))

```

Comment:
unfortunately we must still support this, since the GUI at the moment assumes that the modelpart name is unique

Reply:
Why GUI? I don't know how this (the Model) is used from GiD

Since this is a new function I am not breaking anything, so why not only using the "correct" way?